buzzwindow.cpp

```diff
@@ -137,7 +147,18 @@ int32 TimerThread(void *data)
 		snooze(1000000);
 		T->AddTime(1000000); //+1 sec
 		TimeString = "";
-		TimeString << ((uint32)T->GetHour()) << ":" << ((uint32)T->GetMinute()) << ":" << ((uint32)T->GetSecond());
+		//h:mm:ss format
+		TimeString << ((uint32)T->GetHour()) << ":";
+		if (T->GetMinute() > 9) {
```

Comment:
The curly brackets here nd below aren't needed. Removing them makes the code appear simpler IMO.